bin/config.js

```diff
@@ -0,0 +1,58 @@
+var path = require("path");
+function loadFromCommandLine(args) {
+  args = args || [];
+  while (args.length > 0) {
+    if (args[0] == "--config") {
+      if (args.length < 2) {
+        throw "Please specify the configuration file (json)";
+      }
+      return require(path.resolve(args[1]));
+    }
+    args = args.slice(1, args.length);
+  }
+}
+
+function loadFromEnvironment(env) {
+  env = env || {};
+  var options = {};
+  if (env.PARSE_SERVER_OPTIONS) {
+
+    options = JSON.parse(env.PARSE_SERVER_OPTIONS);
+
+  } else {
+
+    options.databaseURI = env.PARSE_SERVER_DATABASE_URI;
+    options.cloud = env.PARSE_SERVER_CLOUD_CODE_MAIN;
+    options.collectionPrefix = env.PARSE_SERVER_COLLECTION_PREFIX;
```

Comment:
This could be app-specific, allowing multiple apps to share the same database...